README.rst

```diff
@@ -0,0 +1,77 @@
+Silex Skeleton
```

Comment:
github wyswietla README.md wiec ten plik mozna spokojnie usunac

Reply:
ok, czyli mogę go po prostu dorzucić do globalnego .gitignore?

Reply:
a potrzebny ci ten plik do czegoś?

Reply:
Nie, chodziło mi o to że jeszcze nie jestem pewien co od razu wrzucać do globalnego, a co do gitignore w poszczególnych projektach.

Reply:
to mozna .idea jest potrzebne Ci do pracy w PhpStormie plik readme.rst raczej nie. Wszystko co nie jest Ci potrzebne nalezy usunac.

Reply:
plik nadal jest w repo mowilem zeby go usunac ;p

Reply:
jak go usuniesz z repo to mozesz zmergowac PR